README.md

```diff
@@ -1,6 +1,6 @@
 # Golang web analytics for Snowplow
 
-[![actively-maintained]][tracker-classificiation] [![Build Status][travis-image]][travis] [![Coveralls][coveralls-image]][coveralls] [![Goreport][goreport-image]][goreport] [![Release][release-image]][releases] [![GoDoc][godoc-image]][godoc] [![License][license-image]][license]
+[![actively-maintained]][tracker-classification] [![Build Status][travis-image]][travis] [![Coveralls][coveralls-image]][coveralls] [![Goreport][goreport-image]][goreport] [![Release][release-image]][releases] [![GoDoc][godoc-image]][godoc] [![License][license-image]][license]
```

Comment:
Can also likely fix the Travis badge as we have not used Travis on this repo in a veryyyyy long time.